server/apps/immich/src/api-v1/asset/dto/create-asset.dto.ts

```diff
@@ -22,9 +23,29 @@ export class CreateAssetDto {
   @IsNotEmpty()
   isFavorite!: boolean;
 
+  @IsOptional()
```

Comment:
Do we forget to run OpenAPI generator in this PR?

Reply:
Looks like I did, yeah.